network-area-diagram-viewer/src/main/java/com/powsybl/nad/viewer/MainViewController.java

```diff
@@ -172,7 +191,7 @@ protected Task<Network> createTask() {
                     return new Task<>() {
                         @Override
                         protected Network call() {
-                            return Importers.loadNetwork(file.toPath());
+                            return Network.read(file.toPath());
```

Comment:
you need to change the pom to have the 5.0.0-RC1 importers

Reply:
updated